wfc3tools/meta.yaml

```diff
@@ -26,7 +26,7 @@ requirements:
     - scipy
     - python x.x
 source:
-    git_tag: 1.3.1
+    git_tag: 1.3.3
```

Comment:
The "policy" I wrote in "Github and You" says you need to omit the `v` prefix while tagging, but it's more of uniformity/housekeeping issue than anything else. If we use `v` we then need to define the version twice in the recipe.

I need to update `wfc3tools` recipe to match the latest format. Take a look at https://github.com/astroconda/astroconda-contrib/blob/master/acstools/meta.yaml

^ that's what this should look like. I can take care of converting it tomorrow.